compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -1181,6 +1192,7 @@ unsafe extern "C" {
     pub(crate) safe fn LLVMIsGlobalConstant(GlobalVar: &Value) -> Bool;
     pub(crate) safe fn LLVMSetGlobalConstant(GlobalVar: &Value, IsConstant: Bool);
     pub(crate) safe fn LLVMSetTailCall(CallInst: &Value, IsTailCall: Bool);
+    pub(crate) fn LLVMRustSetTailCallKind(CallInst: &Value, Kind: TailCallKind);
```

Comment:
this one can be made `safe`, the caller cannot violate anything that would cause UB